CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-## [unreleased]
+## [17.2.2]
```

Comment:
we discussed offline that for the release notes we can just call it 17.2.2. you think we should be explicit that its 17.2.2_2017?